gulpfile.js

```diff
@@ -1,16 +1,17 @@
 const {series, parallel, task} = require('gulp');
 const allTheThings = require('require-dir')('./gulp/tasks');
 const {
-    clean, copy, development, production, favicon, html, scsslint, styles,
-    humans, images, eslint, scripts, settings, precache, templates, webpack,
-    watch
+  clean, copy, development, production, favicon, scsslint, styles,
+  humans, images, eslint, scripts, settings, precache, templates, webpack,
+  watch, devHTML, distHTML
 } = Object.assign({}, ...Object.values(allTheThings));
 
+
 const beforeWebpack = series(
     clean,
     parallel(
         copy,
-        series(favicon, html),
```

Comment:
@nathanstitt what happened here?

Reply:
there used to be only one `html` step, but I replaced it with `devHTML` and `distHTML`, the dist variant adds the hashes.